packages/gatsby-plugin-sass/src/gatsby-node.js

```diff
@@ -8,18 +8,22 @@ exports.onCreateWebpackConfig = (
     useResolveUrlLoader,
     sassRuleTest,
     sassRuleModulesTest,
-    ...sassOptions
+    sassOptions = {},
+    ...sassLoaderOptions
   }
 ) => {
   const { setWebpackConfig } = actions
   const PRODUCTION = stage !== `develop`
   const isSSR = stage.includes(`html`)
 
+  delete sassLoaderOptions.plugins
```

Comment:
does `sass-loader` not like `plugins` field?

Reply:
Nope, it didn't like that :D 

```
Invalid options object. Sass Loader has been initialized using an
options object that does not match the API schema.
 - options has an unknown property 'plugins'. These properties are
valid:
   object { implementation?, sassOptions?, additionalData?,
sourceMap?, webpackImporter? }

File: src/pages/styles.scss
```